_samples/5_concurrency.md

```diff
@@ -2,24 +2,22 @@
 title: Concurrency Model
 short_name: Concurrency
 description: |
-  Crystal uses green threads, called fibers, to achieve concurrency. Fibers communicate with each other using channels, as in Go or Clojure, without having to turn to shared memory or locks.
-read_more: "[Read more about Crystal's concurrency model](https://crystal-lang.org/reference/guides/concurrency.html)"
+  Crystal uses green threads, called fibers, to achieve concurrency.
+  Fibers communicate with each other via channels without having to turn to shared memory or locks ([CSP](https://www.wikiwand.com/en/Communicating_sequential_processes)).
+read_more: "[Read more about concurrency](https://crystal-lang.org/reference/guides/concurrency.html)"
 ---
 ```crystal
 channel = Channel(Int32).new
-total_lines = 0
 
-sites = ["http://www.example.com", "http://info.cern.ch/"]
-
-sites.each do |site|
+3.times do |i|
   spawn do
-    channel.send site.size
+    3.times do |j|
```

Comment:
Originally, the idea was to have something that really used concurrency, that's why it fetched websites (before it was fetching the sites). It'd be nice if we can think of an example in that spirit.

Reply:
Yeah, that would be great. Fetching random HTTP requests isn't very self-contained though.

Perhaps we could consider both an HTTP server and client in one program? That would be another HTTP example though. And the HTTP overhead shoudn't be necessary to showcase concurrency.

Reply:
We can leave it here and change it later if we find a better example. I agree the HTTP one might be too much. Maybe one reading files?